### PR TITLE
#45 add contact form tests

### LINKED ISSUE #45: Write tests for the contact form
Currently, the frontend portions of the contact form do not have any tests. We should have tests for all the functions in the frontend.

### PR DESCRIPTION
**Pull request by joeySchentrup**
Closes #45

### CODE REVIEW

File: src/modules/contactForm/pages/ContactForm/ContactForm.spec.tsx [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+
+            expect(wrapper.state('activeStep')).toBe(steps.length - 1)
+            expect(props.submitContactForm).toBeCalled()
+        });
+
+        it('should change step on back', () => {
+            wrapper.setState({ activeStep: 1 })
+            wrapper.instance().handleBack()
+
+            expect(wrapper.state('activeStep')).toBe(0)
```
Comment (RyanClementsHax) [inline src/modules/contactForm/pages/ContactForm/ContactForm.spec.tsx]: do you think we should test the edge cases of a `handleBack()` call when the state is `0`? the UI will never get there, but it would provide a nice test to make sure we don't accidentally shoot ourselves in the foot

File: src/modules/contactForm/pages/ContactForm/ContactForm.tsx [hunk 2]
```diff
@@ -153,13 +156,13 @@ function getStepContent(step, state, props, handleChange, handleNext, handleBack
...
             case 0:
-                return <PersonalInformation data={state} handleChange={handleChange} handleNext={handleNext} handleBack={handleBack} />;
+                return <PersonalInformation data={state} handleChange={handleChange} handleNext={handleNext} />;
             case 1:
                 return <ParentsInformation data={state} handleChange={handleChange} handleNext={handleNext} handleBack={handleBack} />;
             case 2:
                 return <Interests data={state} handleChange={handleChange} handleNext={handleNext} handleBack={handleBack} />;
             default:
-                throw new Error('Unknown step');
+                return <p></p>
```
Comment (RyanClementsHax) [inline src/modules/contactForm/pages/ContactForm/ContactForm.tsx]: What are u trying to do here?
